### ISSUE TITLE
How to update WSA here?

### ISSUE THREAD

**Issue by Cr4z33** [LSPosed/MagiskOnWSALocal#110]
I've seen mentioning in the readme the WSA update feature without data loss, but I don't know how to do it.

Can you please explain?

By the way I live in a country (Italy) where WSA is not officially available yet (so no updates through Microsoft Store).

**Comment by Howard20181** [comment]
If you previously installed WSA from Microsoft Store, manually follow [the steps-10-2](https://github.com/LSPosed/MagiskOnWSALocal#text-guide) to backup `userdata.vhdx`.
If you previously installed using this project's script, the script will attempt to preserve the data for upgrade.

**Comment by Cr4z33** [comment]
@Howard20181 so basically I just run install.ps1 again (after having backed up that VHDX file) and that's it? 😁

**Comment by Howard20181** [comment]
Theoretically, you don't need to back up, the script will try to keep the data too. If you are afraid of script errors you can back up manually.

**Comment by Cr4z33** [comment]
Alright thanks gin.

**Comment by Cr4z33** [comment]
Either I am doing something wrong or the script doesn't update my WSA.

What I did:

1. Stopped WSA from running in the background
2. Backed up `userdata.vhdx` at `%LocalAppData%\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalCache\`
3. Ran `install.ps1` and waited for the PowerShell window to end and close
4. Ran WSA setting and went in the Info section
5. It keeps showing v2206.40000.15.0 instead of v2207.40000.8.0.
